traffic_ops/testing/api/utils/utils.go

```diff
@@ -134,10 +144,31 @@ type V5TestData struct {
 	Expectations  []CkReqFunc
 }
 
+type clientSession interface {
+	v3client.Session | v4client.Session | v5client.Session
```

Comment:
Can probably remove `v3client.Session` since not in use by v3?

Reply:
True, removed in 49a7d55de0